test/mocha-fixtures.js

```diff
@@ -12,7 +12,7 @@ let mongoinstance;
 // a replset-instance for running repl-set tests
 let mongorreplset;
 
-// decide wheter to start a in-memory or not
+// decide whether to start an instance or not
```

Comment:
```suggestion
// decide whether to start an instance or use an in-memory database
```

Reply:
it is actually more correct the way it is, because MMS is set to starts a `wiredTiger` (disk stored) and not a `ephermeralForTest` (inmemory) database engine

Reply:
"not" does not give so much information. Can we make that comment a little bit more elaborate?

Reply:
it sounds descriptive to me, but sure, to me it reads `whether to (do thing) or not`

Reply:
should be resolved with fddf23ed4903b7115d29e689553cebc517af5c84, is that better?

Reply:
LGTM, thanks. 👍 
It probably is clear to you and would be clear to me if I dig a bit more and understand more about the context, but I currently don't, and the comment's purpose is to make giving me context easier.

So I'm assuming that if we don't start an instance, we'll start something else. and "not" does not tell much about that thing.